contracts/Overtime/LiquidityPool/SportsAMMV2LiquidityPool.sol

```diff
@@ -152,9 +166,17 @@ contract SportsAMMV2LiquidityPool is Initializable, ProxyOwned, PausableUpgradea
 
         allocationPerRound[nextRound] += amount;
         totalDeposited += amount;
-
-        if (address(stakingThales) != address(0)) {
-            stakingThales.updateVolume(msg.sender, amount);
+        address stakingThales = addressManager.getAddress("StakingThales");
```

Comment:
This code is almost identical to the one in roundClosing, perhaps refactor it to an internal method

Reply:
done